src/components/tipRecords/TipPreview.vue

```diff
@@ -112,7 +112,7 @@
             <Avatar :address="tip.receiver" />
             <div class="tip-author_name">
               <span class="chain-name">
-                {{ tip.chainName }}
+                {{ tip.ChainName.name }}
```

Comment:
question to the backend: usually types starts with a capital letter 🤷‍♀️

Reply:
The problem is with the new aggregation and sequelize we are using, for join of different models the model is upper case in the result, for some materialized view results its all lowercase, so excuse that confusion

Reply:
fixed now :)